examples/keypoint_detection/keypoint_detection/evaluator.py

```diff
@@ -49,17 +50,17 @@ def compute_test_sample_metrics_single(
             return TestSampleMetrics(match_type="failure_to_detect")
 
         normalization_factor = ground_truth.normalization_factor
-        Δ_left_ear, norm_Δ_left_ear = compute_distances(
+        Δ_nose, norm_Δ_nose = compute_distances(
```

Comment:
Was the metrics incorrect before, i.e. value and name/meaning mismatch?

Reply:
I believe so. We don't have an example of this up on any of our main tenants, so I wasn't able to verify this.